schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals(
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOL,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(60)
 );
```

Comment:
- [ ] Let's add a new column called species with a string type to the Animals tables as requested in the project requirements. Also make sure this SQL command is mentioned in `schema.sql` file.
`Hint`: you can use the following command: `ALTER TABLE animals ADD species varchar(255);`